java/openjdk10/Portfile

```diff
@@ -0,0 +1,53 @@
+PortSystem 1.0
```

Comment:
I'd put the editor modeline at the top.  See: https://guide.macports.org/#development.creating-portfile

Reply:
Also, you should indent 1.0 the same way as the rest.